### ISSUE TITLE
Option to exclude tokens and trivia (Syntax Tree)

### ISSUE THREAD

**Issue by alrz** [ashmind/SharpLab#126]
useful when you only care about actual syntax nodes.

**Comment by ashmind** [comment]
Makes sense, but also not sure UI complexity brought by an extra option is outweighted by the gain.
I'll set it to "needs votes" for now, then decide based on 👍s.
